model/src/test/scala/com/lynbrookrobotics/potassium/model/examples/FallingMass.scala

```diff
@@ -0,0 +1,41 @@
+package com.lynbrookrobotics.potassium.model.examples
+
+import breeze.linalg.{DenseMatrix, DenseVector}
+import breeze.plot._
+import com.lynbrookrobotics.potassium.model.StateSpace
+
+object FallingMass extends App {
+  val NumIterations = 1000
+
+  val times = new Array[Double](NumIterations)
+  val positions = new Array[Double](NumIterations)
+
+  val Mass = 1 // kg
+  val GravityForce = Mass * 9.8 // N
```

Comment:
Why is this not using squant units?
nit: first letter should be lower case for a variable name